src/app/personside/infotabs/meldinger/traadvisning/Enkeltmelding.tsx

```diff
@@ -56,7 +62,14 @@ function EnkeltMelding(props: Props) {
         <div className="snakkeboble_ikoner">
             <Snakkeboble pilHoyre={fraNav} ikonClass={fraNav ? 'nav-ikon' : 'bruker-ikon'}>
                 <SnakkebobleWrapper>
-                    <Element>{topptekst}</Element>
+                    <Topptekst>
+                        <Element>{topptekst}</Element>
+                        {props.melding.status === LestStatus.Lest ? (
```

Comment:
Denne kan du godt dra ut i en konstant. I litt større komponenter blir det fort vanskelig å lese sånne inline statements

Reply:
<img width="724" alt="Screen Shot 2019-09-26 at 2 40 09 PM" src="https://user-images.githubusercontent.com/11627538/65688787-f6e78000-e06b-11e9-9ac7-635ffbc1c261.png">

Reply:
Bilde for å illustrere hvordan det ble